### PR TITLE
Add my Overlay

### PR DESCRIPTION
**Pull request by nelsongraca**
I have had this overlay for quite some time, I decided to finally ask to include it on the official list.

### CODE REVIEW

File: files/overlays/repositories.xml [hunk 1]
```diff
@@ -5294,6 +5294,18 @@
     <source type="git">git+ssh://git@github.com/xartin/gentoo-overlay.git</source>
     <feed>https://github.com/xartin/gentoo-overlay/commits/master.atom</feed>
   </repo>
+  <repo quality="experimental" status="unofficial">
+    <name>nelson-graca</name>
```
Comment (mgorny) [inline files/overlays/repositories.xml]: The same name must be specified in profiles/repo_name

Comment (nelsongraca) [reply files/overlays/repositories.xml]: Yea, I saw it on semaphore, should be fixed by now, I changed the name on the overlay side this should be the proper one.

Comment (nelsongraca) [reply files/overlays/repositories.xml]: Can the build be re-triggered or do I need to add a commit to the PR?

Comment (mgorny) [reply files/overlays/repositories.xml]: Done and looks good now.

**Pull request closed without merging**